patterns/full-width-cover-banner.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Title: Full-width Cover Banner
+ * Slug: twentytwentyfive/full-width-cover-banner
+ * Categories: banner, call-to-action, media, featured
+ * Description: A full-width cover banner that can be applied to a page or it can work as a single landing page.
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twenty_Five
+ * @since Twenty Twenty-Five 1.0
+ */
+
+?>
+<!-- wp:cover {"url":"<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/coming-soon-bg-image.webp","alt":"Photo of a field full of flowers, a blue sky and a tree.","dimRatio":30,"overlayColor":"contrast","isUserOverlayColor":true,"minHeight":100,"minHeightUnit":"vh","align":"full","style":{"elements":{"link":{"color":{"text":"var:preset|color|accent-1"}},"heading":{"color":{"text":"var:preset|color|accent-1"}}},"spacing":{"padding":{"right":"var:preset|spacing|50","left":"var:preset|spacing|50","top":"var:preset|spacing|50","bottom":"var:preset|spacing|50"},"margin":{"top":"0","bottom":"0"}}},"textColor":"accent-1","layout":{"type":"constrained"}} -->
+<div class="wp-block-cover alignfull has-accent-1-color has-text-color has-link-color" style="margin-top:0;margin-bottom:0;padding-top:var(--wp--preset--spacing--50);padding-right:var(--wp--preset--spacing--50);padding-bottom:var(--wp--preset--spacing--50);padding-left:var(--wp--preset--spacing--50);min-height:100vh"><span aria-hidden="true" class="wp-block-cover__background has-contrast-background-color has-background-dim-30 has-background-dim"></span><img class="wp-block-cover__image-background" alt="Photo of a field full of flowers, a blue sky and a tree." src="<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/coming-soon-bg-image.webp" data-object-fit="cover"/>
+	<div class="wp-block-cover__inner-container">
+		<!-- wp:heading {"textAlign":"center","level":1,"fontSize":"small"} -->
+		<h1 class="wp-block-heading has-text-align-center has-small-font-size">Event</h1>
```

Comment:
I hadn't noticed that the pill style PR had been merged already. Can we replace it with it?

```
<!-- wp:group {"layout":{"type":"flex","orientation":"vertical","justifyContent":"center"}} -->
<div class="wp-block-group"><!-- wp:heading {"textAlign":"center","level":1,"className":"is-style-pill","fontSize":"small","borderColor":"accent-1"} -->
<h1 class="wp-block-heading has-text-align-center is-style-pill has-border-color has-accent-1-border-color has-small-font-size">Event</h1>
<!-- /wp:heading --></div>
<!-- /wp:group -->
```

Reply:
Absolutely 🫡